### ISSUE TITLE
FOR HELP

### ISSUE THREAD

**Issue by Westbrone** [lucasmansilla/ACRN_Chest_X-ray_IA#34]
Hello, your code is written very well, but because there is no pre-training model of the autoencoder, when I was training the model, the loss remained at around 0.49 in the end. Is it because of the pre-training model?
